pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -122,6 +124,15 @@ func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextM
 		return nil, kerrors.NewAggregate(err)
 	}
 
+	// resolve extended arguments
+	// TODO: this should be done in config validation (along with the above) so we can provide
+	// proper errors
+	schedulerserver := scheduleroptions.NewSchedulerServer()
```

Comment:
thanks, can you make an equivalent to `TestCMServerDefaults` that smacks us when the defaults change so we can decide if any need overriding?